pkg/storage/postgres/postgres.go

```diff
@@ -677,8 +672,8 @@ func (p *Postgres) ReadChanges(
 	if objectTypeFilter != "" {
 		sb = sb.Where(sq.Eq{"object_type": objectTypeFilter})
 	}
-	if opts.From != "" {
-		token, err := sqlcommon.UnmarshallContToken(opts.From)
+	if options.Pagination.From != "" {
```

Comment:
can `Pagination` be nil? in that case won't this create a panic?

Reply:
I wondered about this as well. If the options contain no `Pagination` value, e.g.,

`readChangesOpts := storage.ReadChangesOptions{}`

Then the zero/default values are:
`opts.Pagination.From == ''`
`opts.Pagination.PageSize == 0`

Checking for nil `Pagination` won't compile given that `options` is not a pointer.

Given that, if there is a more idiomatic Go way of ensuring nil safety I'm happy to change it :)